docs-2.0/1.introduction/2.data-model.md

```diff
@@ -0,0 +1,19 @@
+# Data model
+
+This document gives some introduction to Nebula Graph data model. A data model is a model that organizes data and specify how they are related to one another.
+
+## Data structures
+
+Nebula Graph data model uses five data structures to store data. They are vertices, edges, properties, tags and edge types.
+
+- **Vertices**: Vertices are used to store entities.
+  - In Nebula Graph, vertices are identified with vertices identifiers (i.e. `VID`). The `VID` must be unique in the same graph space.
+  - A vertex must have at least one tag.
+- **Edges**: Edges are used to connect vertices. An edge is a connection or a behavior between two vertices.
+  - An edge is identified uniquely with a source vertex, an edge type, a rank value, and a destination vertex.
+  - Edges are directed. `->` identifies the directions of edges. Edges can be traversed in either direction.
+  - An edge must have one and only one edge type.
+  - Rank value is an immutable user-assigned 64-bit signed integer. It identifies the edges with the same edge type between two vertices. Edges with higher rank values comes first. The default rank value is zero.
+- **Properties**: Properties are key value pairs. Both vertices and edges are containers for properties.
+- **Tags**: Tags are used to categorize vertices. Vertices that have the same tag are collected into a group.
```

Comment:
**Tags**: Tags are used to categorize vertices. Vertices that have the same tag are collected into a group （to share the same schema）.

Reply:
I don't want to introduce Schema in this doc. Maybe in a new doc.

Reply:
Tags: Tags are used to categorize vertices. Vertices that have the same tag are collected into a group （to share the same definition of properties.）.

Otherwise, it is confusing -- what do you mean by group? is it a label?